example/themes/original/src/components/BusinessPreorder/index.tsx

```diff
@@ -9,9 +9,9 @@ import FastImage from 'react-native-fast-image'
 import CalendarStrip from 'react-native-calendar-strip'
 import { BusinessMenuList } from '../BusinessMenuList'
 import Spinner from 'react-native-loading-spinner-overlay'
-import { BusinessPreorderParams } from '../../types'
+// import { BusinessPreorderParams } from '../../types'
```

Comment:
You can't do that, if is necessary you need to update the main function, but first you can't add comments and second you can't create custom functions for this case.